app/javascript/components/supplemental_application/sections/preferences/LiveOrWorkInSanFranciscoPanel.js

```diff
@@ -25,12 +25,14 @@ const getLWPrefProofTypeOptions = (individualPrefName) => {
 class LiveOrWorkInSanFranciscoPanel extends React.Component {
   constructor (props) {
     super(props)
+    // I do not think we need a state for this, Fed
```

Comment:
@fddayan I agree we can likely optimize here. In the spirit of keeping PRs focused on story AC, let's look at this area in a more appropriate Pivotal story - specifically this one that deals with Live/Work proof types: https://www.pivotaltracker.com/story/show/160007121. I've put an engineering note in that story to look at this area. Since we've captured your observation in that story's notes, please remove these two comments from this PR. Thanks for looking out for where we can improve!